pyproject.toml

```
@@ -26,7 +26,7 @@ requires = [
     "cachetools==5.2.0",
     "importlib-metadata==4.12.0",
     "evaluate",
-    "datasets==2.6.1",
+    "datasets==2.6.2",
```

we dont need to pin datasets so tightly here..
this should be fine 
```suggestion
    "datasets>=2.6",
```

---

Sure, let's try it.

In general we ought to move toward much, much less pinning.  Right now, it's easy to imagine training setups that would break when dq is installed.

---

yup i definitely agree